### PR TITLE
update micro example

### PR DESCRIPTION
**Pull request by zaunist**
update the example_micro_handler_test

### CODE REVIEW

**Comment by wu-sheng** [comment]
Why so many commits? Please rebase or check out from the latest commit from the master branch again.
We don't want to make the commit log chaotic.

**Comment by zaunist** [comment]
Something went wrong, let me check

**Comment by zaunist** [comment]
@wu-sheng  Please review it.

File: micro/example_micro_handler_test.go [hunk 1]
```diff
@@ -38,7 +38,7 @@ func (g *Greeter) Hello(ctx context.Context, name *string, msg *string) error {
 
 func ExampleNewHandlerWrapper() {
 	//Use gRPC reporter for production
-	r, err := reporter.NewLogReporter()
+	r, err := reporter.NewGRPCReporter("skywalking-oap:11800")
```
Comment (arugal) [inline micro/example_micro_handler_test.go:41]: @zaunist Hi, I think the example should use `LogReporter`, so it was good before :)

**Review by arugal — CHANGES_REQUESTED** [review]

File: micro/example_micro_handler_test.go (hunk 1 above)
Comment (zaunist) [reply micro/example_micro_handler_test.go]: Agree，I already update it

**Comment by zaunist** [comment]
I updated the example and readme, please review it. :blush:

**Comment by arugal** [comment]
> I updated the example and readme, please review it. 😊

Sorry for the late reply :smile: . My point is that the previous example was pretty good, so don't need to change it.

**Pull request closed without merging**